lib/db/versioning.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * @module stardogjs.db.versioning
```

Comment:
Last I knew this was going away as of Stardog 7. Check with Evren to make sure this should even bother being here now